packages/lib/services/plugins/api/JoplinViewsDialogs.ts

```diff
@@ -98,4 +98,12 @@ export default class JoplinViewsDialogs {
 		return this.controller(handle).open();
 	}
 
+	/**
+	 * Toggle on whether to fit the dialog size to the content or not.
+	 * When set to false, the dialog is set to 90vw and 80vh
+	 * @default true
+	 */
+	async fitToContent(handle: ViewHandle, status: boolean) {
```

Comment:
Thanks for the update. It should be `setFitToContent` for consistency with the other properties.

Reply:
Done.